### PR TITLE
squashing bugs and next7days

### PR DESCRIPTION
**Pull request by djoii**

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -6,14 +6,13 @@ const fs = require('fs');
 
 const app = express();
 app.use(bodyParser.json());
-
+app.set('view engine', 'html');
```
Comment (neilfulwiler) [inline server/index.js]: we can remove this if we're just using .json() I think, we probably don't want to use render

File: server/index.js [hunk 2]
```diff
@@ -40,8 +39,9 @@ app.post('/api/todos/create', (req, res) => {
  * there was no error
  */
 function handleError(res, error) {
+  console.log(error)
```
Comment (neilfulwiler) [inline server/index.js]: remove

File: src/App.js [hunk 3]
```diff
@@ -3,6 +3,7 @@ import PropTypes from 'prop-types';
 import Inbox from './Inbox';
 import Today from './Today';
 import Menu from './Menu';
+import Next7Days from './Next7Days';
```
Comment (neilfulwiler) [inline src/App.js]: i don't see this file?

Comment (djoii) [reply src/App.js]: how do i include it in the push?

Comment (neilfulwiler) [reply src/App.js]: I think just git add .

File: src/TodoList.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
```
Comment (neilfulwiler) [inline src/TodoList.js]: did you mean to add this file too?

Comment (djoii) [reply src/TodoList.js]: i dunno what that is?

Comment (neilfulwiler) [reply src/TodoList.js]: I don't think its used, `git rm src/TodoList.js`

**Pull request merged**